common/src/Model/Brush.cpp

```diff
@@ -894,29 +894,12 @@ namespace TrenchBroom {
                 result = std::move(nextResults);
             }
 
-            // create Brushes and discard any that fail to create
-            std::vector<Brush> resultBrushes;
-            std::optional<BrushError> firstError;
-
-            resultBrushes.reserve(result.size());
-
-            for (auto&& geometry : result) {
-                createBrush(mapFormat, worldBounds, defaultTextureName, geometry, subtrahends)
-                    .and_then([&](Brush&& brush) {
-                        resultBrushes.push_back(brush);
-                    })
-                    .handle_errors([&](const BrushError& e) {
-                        if (!firstError) {
-                            // let the caller know that there was at least one failed brush
-                            // (probably a harmless microbrush)
-                            firstError = e;
-                        }
-                    });
-            }
-            return std::make_tuple(resultBrushes, firstError);
+            return kdl::collect_results(result, [&](const auto& geometry) {
```

Comment:
This is just `kdl::vec_transform`, right?

Reply:
Yeah, good point. I think I'll just in-line this into collect_values.